### ISSUE TITLE
[BUG] - misleading error message when --upper-bound is used together with --ttl

### ISSUE THREAD

**Issue by mkoura** [input-output-hk/cardano-node#2189]
*Internal*

**Summary**
If `--upper-bound` is used together with `--ttl` when building transaction, the error message makes it seem like the `--upper-bound` option doesn't exist:

```
$ cardano-cli shelley transaction build-raw --ttl 19150 --fee 324429 --out-file test_script_time_locking_from_tx.body --tx-in b0edd77252ee0132de869c47874fc5c2b20191f44bf20b5864b6a09edf7ad1a4#0 --tx-out addr_test1vzhx8649xq8yx69xrykct5300zep8ryje50nygu52pkqscql0396f+1000 --tx-out addr_test1wrqdlyzgtv5t6zukr43zpwg7y8f2yrvac3qrdl38ps2c0kgt4up88+1674571 --lower-bound 101 --upper-bound 17225 --allegra-era
Invalid option `--upper-bound'

Usage: cardano-cli (COMMAND | Byron specific commands | Miscellaneous commands)
```
Same error message is printed for any non-existent option:
```
$ cardano-cli transaction build-raw --fee 324429 --out-file test_script_time_locking_from_tx.body --tx-in b0edd77252ee0132de869c47874fc5c2b20191f44bf20b5864b6a09edf7ad1a4#0 --tx-out addr_test1vzhx8649xq8yx69xrykct5300zep8ryje50nygu52pkqscql0396f+1000 --tx-out addr_test1wrqdlyzgtv5t6zukr43zpwg7y8f2yrvac3qrdl38ps2c0kgt4up88+1674571 --not-exist 17225 --allegra-era
Invalid option `--not-exist'

Usage: cardano-cli (COMMAND | Byron specific commands | Miscellaneous commands)
```

**Steps to reproduce**
1. prepare script address with funds
2. try to spend from the script address using both `--ttl` and `--upper-bound` options
3. See error

**Expected behavior**
An error message that indicates that `--ttl` and `--upper-bound` options cannot be used together.

**System info (please complete the following information):**
- OS: Fedora
- Version 32
- Node version 196ba716c425a0b7c75741c168f6a6d7edaee1fc

**Comment by Jimbo4350** [comment]
The deprecated command `--ttl` was not hidden and probably led to incorrectly specifying `--ttl` and `--upper-bound` which are the same thing. I've hidden the deprecated commands in #2190.

**Comment by dcoutts** [comment]
Fixed in #2190.
